src/pages/SetupPlot.vue

```diff
@@ -109,71 +105,65 @@ export default defineComponent({
       debounce(async (val) => {
         console.log(val)
         if (this.plotDirectory == this.defaultPath) return (this.validPath = true)
-        this.validPath = await util.native.dirExists(val)
+        this.validPath = await native.dirExists(val)
         if (this.validPath) await this.updateDriveStats()
+        return
       }, 500)
     )
   },
   computed: {
     chartData(): any {
```

Comment:
This not only need a proper type, but unless it is an object with descriptive keys, it also needs a comment explaining what is being returned from here so that it is readable from function signature + its comment.

Reply:
anon function inside debouce function so returns null, only added a return because linting error

Reply:
Hold on resolving this, I don't quite understand what you said. This function returns an array with 3 elements, what `null` are you talking about?

Reply:
Sorry I'm looking at the function above the one you are talking about. I have resolved the issue in the latest commit.